### DISCUSSION TITLE
Full Screen Lottie

### DISCUSSION THREAD

**Discussion by fantasy-dev** [timbru31/cordova-plugin-lottie-splashscreen#478] [General]
I added a custom lottie made by me and it doesnt stretch to full screen leaving me with white padding at the top and bottom. Can i stretch it with any functionality or do you know the "go-to" lottie ratio sizes? Thanks!

**Comment by timbru31** [comment]
Are you facing the issue on iOS, Android or both? For Android you can use different scaling modes: https://developer.android.com/reference/android/widget/ImageView.ScaleType

Also, have you seen the fullscreen option?
```xml
    <preference name="LottieFullScreen" value="true" />
```

**Reply by jakubperdoch** [reply]
On iOS even after I added this. Still displaying in bottom corner small splash screen

**Reply by golfacross** [reply]
I have the same issue... 

`preferences: {
			LottieFullScreen: "true",
			LottieHideAfterAnimationEnd: "true",
			LottieAnimationLocation: "public/assets/splash/splash.json", }`

Any solution to this?

**Reply by diegoanvilla** [reply]
Same here, any solution?

**Reply by mayerlench** [reply]
Same issue

**Reply by papabaab** [reply]
Same problem! The animation item displays at the bottom right of the screen in iOS.
